android/README.md

```diff
@@ -9,7 +9,20 @@
 
 Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+The Ping SDK for Android enables you to integrate your SPA with our services.
+There are two methods of UI integration:
+- Embedded - With this option, each app has to have its own login User Interface (UI).
+  When a user attempts to log in to your application or site, they authenticate directly within the currently used application instead of being directed to a centralized web application to authenticate the user.
+- OIDC Login - Often referred to as centralized login,
+  with this option you reuse a central UI
+  (such as hosted pages for PingOne Advanced Identity Cloud or the Login Widget for PingOne/PingOne DaVinci)
+  or your own web application for login requests in multiple apps and sites.
+  When a user attempts to log in to your application or site, they are redirected to a central login UI. After the user authenticates, they are redirected back to your application or site.
+
+The sample apps focus on embedded login,
```

Comment:
It's a bit of a mix really, perhaps just say something like "We provide samples the demonstrate both methods."